include/GafferScene/Instancer.h

```diff
@@ -99,13 +147,40 @@ class GAFFERSCENE_API Instancer : public BranchCreator
 		Gaffer::BoolPlug *encapsulateInstanceGroupsPlug();
 		const Gaffer::BoolPlug *encapsulateInstanceGroupsPlug() const;
 
+		Gaffer::BoolPlug *generateSeedsPlug();
+		const Gaffer::BoolPlug *generateSeedsPlug() const;
+
+		Gaffer::StringPlug *seedVariablePlug();
+		const Gaffer::StringPlug *seedVariablePlug() const;
+
+		Gaffer::IntPlug *numSeedsPlug();
+		const Gaffer::IntPlug *numSeedsPlug() const;
+
+		Gaffer::IntPlug *seedsScramblePlug();
+		const Gaffer::IntPlug *seedsScramblePlug() const;
+
+		Gaffer::BoolPlug *seedsPassthroughAllIdsPlug();
+		const Gaffer::BoolPlug *seedsPassthroughAllIdsPlug() const;
```

Comment:
I've gone with singular, and used "possibilities" instead of "count" or "variations" (  "variations" would work, but we use "variations" to describe "actual number of different things being output", so it would be confusing to use the same word to describe the possible range of outputs ).  This leaves me with:
seedEnabled
seedVariable
seedPossibilities
seedPermutation
rawSeed